keras_cv/layers/object_detection/anchor_generator.py

```diff
@@ -70,7 +70,7 @@ class AnchorGenerator(keras.layers.Layer):
     Input shape: an image with shape `[H, W, C]`
     Output: a dictionary with integer keys corresponding to each level of the feature
         pyramid.  The size of the anchors at each level will be
-        `(H/strides * W/strides * K, 4)`.
+        `(H/strides * W/strides * anchors_per_location, 4)`.
```

Comment:
maybe also comment under this line:
`anchors_per_location=len(scales) * len(aspect_ratios)`
?

Reply:
Updated - thanks!